--- setup.py
+++ setup.py
@@ -52,7 +52,7 @@
 copt =  {
     'gcc' : ['-fopenmp','-O3','-ffast-math'],
     'icc' : ['-openmp','-O3'],
-    'clang' : ['-O3','-ffast-math'],
+    'clang' : ['-fopenmp','-O3','-ffast-math'],
     #'clang w/ OpenMP' : ['-fopenmp=libomp','-O3','-ffast-math'],
     'clang w/ OpenMP' : ['-fopenmp','-O3','-ffast-math'],
     'unknown' : [],
@@ -60,7 +60,7 @@
 lopt =  {
     'gcc' : ['-fopenmp'],
     'icc' : ['-openmp'],
-    'clang' : [],
+    'clang' : ['-fopenmp'],
     'clang w/ OpenMP' : ['-fopenmp'],
     'unknown' : [],
 }
