--- setup.py
+++ setup.py
@@ -55,14 +55,14 @@
     'clang' : ['-O3','-ffast-math'],
     #'clang w/ OpenMP' : ['-fopenmp=libomp','-O3','-ffast-math'],
     'clang w/ OpenMP' : ['-fopenmp','-O3','-ffast-math'],
-    'unknown' : [],
+    'unknown' : ['-fopenmp','-O3','-ffast-math', '-I%s/include' % os.environ['PREFIX']],
 }
 lopt =  {
     'gcc' : ['-fopenmp'],
     'icc' : ['-openmp'],
     'clang' : [],
     'clang w/ OpenMP' : ['-fopenmp'],
-    'unknown' : [],
+    'unknown' : ['-fopenmp',  '-L%s/lib' % os.environ['PREFIX']],
 }

 if "--debug" in sys.argv:
